Misc/Positions_Area Positions 3 UK.txt

```diff
@@ -11,23 +11,13 @@ London Control S10:London Control:128.120:LNE:L:LON-10:CTR:-:-:0301:0377::
 London Control S10:London Control:128.120:LNE:L:LON-11:CTR:-:-:0301:0377::
 London Control:London Control:126.770:LWN:L:LON:CTR:-:-:::N053.39.29.755:W001.07.20.927:N051.18.21.496:W004.09.51.334
 ;West
-London Control S5:London Control:133.600:L35:L:LON:CTR:-:-:0301:0377::
-London Control S5:London Control:133.600:L35:L:LON-WU:CTR:-:-:0301:0377::
-London Control S5:London Control:133.600:L35:L:LON-WN:CTR:-:-:0301:0377::
-London Control S5:London Control:133.600:L35:L:LON-W5:CTR:-:-:0301:0377::
-London Control S5:London Control:133.600:L35:L:LON-35:CTR:-:-:0301:0377::
-London Control S8:London Control:129.370:L5:L:LON:CTR:-:-:0301:0377::
-London Control S8:London Control:129.370:L5:L:LON-WN:CTR:-:-:0301:0377::
-London Control S8:London Control:129.370:L5:L:LON-WW:CTR:-:-:0301:0377::
-London Control S8:London Control:129.370:L5:L:LON-W5:CTR:-:-:0301:0377::
-London Control S8:London Control:129.370:L5:L:LON-W8:CTR:-:-:0301:0377::
-London Control S23:London Control:134.750:L23:L:LON:CTR:-:-::
-London Control S23:London Control:134.750:L23:L:LON-WL:CTR:-:-::
-London Control S23:London Control:134.750:L23:L:LON-WE:CTR:-:-::
-London Control S23:London Control:134.750:L23:L:LON-23:CTR:-:-::
-London Control S9:London Control:132.950:L6:L:LON:CTR:-:-::
-London Control S9:London Control:132.950:L6:L:LON-WS:CTR:-:-::
-London Control S9:London Control:132.950:L6:L:LON-W6:CTR:-:-::
+LON_WR_CTR:London Control:134.750:LWR:L:LON:CTR:-:-:0301:0377:N051.37.36.130:W003.28.16.642
+LON_WB_CTR:London Control:133.600:LWB:L:LON:CTR:-:-:0301:0377:N051.37.36.130:W003.28.16.642
+LON_WT_CTR:London Control:129.370:LWT:L:LON:CTR:-:-:0301:0377:N051.37.36.130:W003.28.16.642
+LON_WP_CTR:London Control:133.220:LWP:L:LON:CTR:-:-:0301:0377:N051.37.36.130:W003.28.16.642
+LON_WL_CTR:London Control:132.950:LWL:L:LON:CTR:-:-:0301:0377:N051.37.36.130:W003.28.16.642
+LON_WH_CTR:London Control:127.700:LWH:L:LON:CTR:-:-:0301:0377:N051.37.36.130:W003.28.16.642
+LON_WX_CTR:London Control:128.820:LWX:L:LON:CTR:-:-:0301:0377:N051.37.36.130:W003.28.16.642
```

Comment:
I think this would be a whole lot easier for me to understand with numbers to be honest. I also think it's far easier to be referring to numbers on a diagram, given that we should always have a diagram when using these positions. It's quite difficult cross checking the ownership when I'm not sure what these letters refer to. 

Also, could the start of the lines be left as 'London Control SX' as well, so these are clearer?

Will we need to define additional lines for each in the case that the relief callsign may need to use a hyphen, rather than an underscore?

Reply:
As above ^

Could you come up with a schematic for the intended top-down coverage of each sector by controller position, and too a list of airfields per sector? This should make the review a little easier.

What are your thoughts on using numbers for our west splits? I personally think they would be less confusing than WR, WB, WT, WP, WL, WH, WX... R? Brecon? T? Pembroke? Low? High? X?

Reply:
Prefix that I have briefly discussed briefly with you already Harry.

Numbers don't allow for relief callsigns very well. The number could (perhaps) be the relief callsign, but we probably need a letter anyway as the primary. I suppose it depends on how we name these sectors. If we refer to them by number, the first callsign should be a number. If we give them a name, the first callsign should be the name.

e.g.
"Brecon"
LON_WB_CTR <=> LON_5_CTR

Reply:
Here is an image depicting the splits, to aid understanding/discussion.

![New West Sectors](https://user-images.githubusercontent.com/6871785/93251294-e738de00-f78b-11ea-83e4-d796c02cbeb2.png)